### PR TITLE
Vm disk resize

### PR DESCRIPTION
**Pull request by gekorob**
Introduce disk resize for VMs

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1671330

### CODE REVIEW

File: spec/models/manageiq/providers/redhat/infra_manager/ovirt_services/v4_spec.rb [hunk 1]
```diff
@@ -328,7 +328,7 @@
 
       subject(:reconfigure_vm) { ems.vm_reconfigure(vm, :spec => spec) }
 
-      it 'calls the add command with mandatory params' do
+      it 'calls the remove command with mandatory params' do
```
Comment (agrare) [inline spec/models/manageiq/providers/redhat/infra_manager/ovirt_services/v4_spec.rb]: Ha whoops :)

Comment (gekorob) [reply spec/models/manageiq/providers/redhat/infra_manager/ovirt_services/v4_spec.rb]: Yes, It was my previous mistake... sorry :-(

**Comment by simaishi** [comment]
Changing to `jansa/yes?` to let this go through triage process.

**Comment by mwperina** [comment]
OK, there is customer ticket behing this RFE, so we will be doing backport to Ivanchuk, so marking as that. The only difference is, that Jansa backport should be clean, while Ivanchuk will require special PR

**Comment by simaishi** [comment]
@mwperina Thank you for clarification. Please update target release on the BZ, as the BZ currently shows this RFE is not targeted for ivanchuk. Thanks!

**Comment by simaishi** [comment]
Jansa backport details:

```
$ git log -1
commit 51058976fe1e4601193369fe015bb84d96a0aeac
Author: Adam Grare <agrare@redhat.com>
Date:   Tue Mar 17 14:03:01 2020 -0400

    Merge pull request #483 from gekorob/rfe/vm_reconf_disk_resize

    Vm disk resize

    (cherry picked from commit f7ef9b18db12988dd71f47936af35ab160a326d7)
```

**Comment by simaishi** [comment]
Backported to ivanchuk via https://github.com/ManageIQ/manageiq-providers-ovirt/pull/496

**Pull request merged**